pandas-stubs/core/frame.pyi

```diff
@@ -732,11 +732,7 @@ class DataFrame(NDFrame, OpsMixin, _GetItemHack):
         axis: Axis | None = ...,
         level: Level | None = ...,
         copy: _bool = ...,
-        fill_value=...,
-        method: FillnaOptions | None = ...,
-        limit: int | None = ...,
-        fill_axis: Axis = ...,
-        broadcast_axis: Axis | None = ...,
+        fill_value: Hashable | None = ...,
```

Comment:
some methods use `Hashable` for `fill_value`, others `Scalar`, I'm not sure. anyway for now in this PR I'm just going with what's in pandas